src/app/app.permissions.js

```diff
@@ -0,0 +1,14 @@
+angular.module('librarium').run(function (PermRoleStore, Session, UserPersistence) {
+  // logged in users
+  PermRoleStore.defineRole('USER', []);
```

Comment:
implement this role as well, it should use the same Session service to test if user is logged in